packages/angular_devkit/build_angular/src/builders/dev-server/tests/options/port_spec.ts

```diff
@@ -40,23 +40,9 @@ describeBuilder(serveWebpackBrowser, DEV_SERVER_BUILDER_INFO, (harness) => {
     });
 
     it('uses default port (4200) when not present', async () => {
-      harness.useTarget('serve', {
-        ...BASE_OPTIONS,
-        // Base options set port to zero
-        port: undefined,
-      });
-
-      const { result, response, logs } = await executeOnceAndFetch(harness, '/');
-
-      expect(result?.success).toBeTrue();
-      expect(getResultPort(result)).toBe('4200');
-      expect(await response?.text()).toContain('<title>');
+      const result = await _defaultOptions({ browserTarget: 'test:build' });
```

Comment:
This change bypasses the end user facing logic.


What we can do in this case to valid default port is the validate that the dev-server started on port 4200, if that fails check for the error messages that port 4200 is in use.

Reply:
I'll try that so we don't have to disable parallel tests 👍